tests/crunchydb_dev_provision.robot

```diff
@@ -9,17 +9,18 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
 Test Setup          Given Setup The Test Case
 Test Teardown       Tear Down The Test Case
 Suite Teardown      Tear Down The Test Suite
+Force Tags          CLI     crunchy
```

Comment:
Why do you put CLI tagging here? Isn't it a UI test?

Reply:
Thanks for pointing that, It was by mistake and updated now.